src/v/coproc/ntp_context.h

```diff
@@ -34,6 +34,8 @@ struct ntp_context {
         model::offset last_acked{model::model_limits<model::offset>::min()};
     };
 
+    using offset_tracker = absl::btree_map<script_id, offset_pair>;
```

Comment:
sorry, i know this patch doesn't actually make the type choice decision: but do you need ordering on the key (or, why btree_map)?

Reply:
I don't need ordering on the key, but when I introduced this type I figured to use btree_map to save space as I could envision this map becoming quite large